setup.py

```diff
@@ -13,7 +13,8 @@
 
 install_requires = [
     "neuromation>=20.9.24",
-    "yarl",
+    "yarl==1.5.1",
+    "multidict==4.7.6",
```

Comment:
why don't we just use the same constraints as [in neuro](https://github.com/neuro-inc/platform-client-python/blob/d02710e36c2709ffa8465ff61cbb58a01d2e9156/setup.py#L23-L24)?
```
        "yarl>=1.5.1",
        "multidict>=4.0",
```

Reply:
`>=1.5.1` would install 1.6.3

Reply:
weird...